espressif/vscode-esp-idf-extension
# Translation using l10n vscode api

## Description

Use `vscode.l10n.t()` for source code translation to reduce translation files and phrases duplication.
Add missing translation from package.json to fix GitHub issue.
Add Portuguese translation.

Fixes #1165 
Fixes #567 

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## Steps to test this pull request

This PR provides a list of translations. You need to switch vscode language to observer commands and text translation.

1. Click on "Configure display language" vscode command. Choose one of our supported languages (Chinese, English, Spanish, Russian).
2. Execute action. Vscode language should be changed and extension commands and text should also change.
3. Observe results. 

- Expected behaviour:
- Extension language change to vscode new language.


## How has this been tested?

Manual Testing

**Test Configuration**:
* ESP-IDF Version: Irrelevant
* OS (Windows,Linux and macOS): macOS

## Checklist
